src/Extensions/Logging/ServiceCollectionExtensions.cs

```diff
@@ -34,14 +34,38 @@ public static ILoggingBuilder AddOmexLogging(this ILoggingBuilder builder)
 			return builder;
 		}
 
+		/// <summary>
+		/// Load the initial logger
+		/// </summary>
+		/// <param name="builder">The extension method argument</param>
+		public static ILoggingBuilder LoadInitialLogger(this ILoggingBuilder builder)
+		{
+			builder.AddConsole();
+			builder.Services.TryAddTransient<IServiceContext, EmptyServiceContext>();
+			builder.Services.TryAddTransient<IExecutionContext, BasicMachineInformation>();
+			builder.Services.TryAddTransient<IExternalScopeProvider, LoggerExternalScopeProvider>();
+			builder.Services.TryAddTransient<IActivityProvider, ReplayableActivityProvider>();
```

Comment:
I don't think we need `IActivityProvider` here, if we do then we should use `SimpleActivityProvider` instead

Reply:
Or alternatively, you could use `AddOmexLogger` instead to make sure that the same logic used in both cases